stdlib/public/runtime/Win32.cpp

```diff
@@ -71,4 +76,61 @@ _swift_win32_copyWideFromUTF8(const char *str) {
   return nullptr;
 }
 
+#pragma mark - DbgHelp library thread-safety
+
+static LazyMutex dbgHelpMutex;
+static HANDLE dbgHelpHandle = nullptr;
```

Comment:
Can we use `INVALID_HANDLE_VALUE` instead of `nullptr`?  It seems more appropriate even though it may make the checked condition more verbose.